pom.xml

```diff
@@ -1001,11 +1004,26 @@
         <artifactId>maven-failsafe-plugin</artifactId>
         <executions>
           <execution>
-            <id>run-integration-tests</id>
+            <id>run-minicluster-integration-tests</id>
             <goals>
               <goal>integration-test</goal>
               <goal>verify</goal>
             </goals>
+            <configuration>
+              <skip>${accumulo.skipMiniClusterInFailsafe}</skip>
+              <groups>${accumulo.miniclusterTests}</groups>
+            </configuration>
+          </execution>
+          <execution>
```

Comment:
I guess I was thinking if a class is marked with more than one category, or is run under both executions. It would clobber the previous report for the same class.